ui/components/layouts/dashboard.js

```diff
@@ -23,7 +23,12 @@ function Layout({ children }) {
       method: 'POST',
     })
     await mutate('/api/users/self', async () => undefined)
-    router.replace('/login')
+
+    window.localStorage.setItem('logout', true)
```

Comment:
Storing this temporary state seems like a band-aid for logic somewhere else – what issue is this fixing – perhaps we can fix that in the `AuthRequired` component

Reply:
If this is for something like #2526 it should be fixed now

Reply:
the reason why we store the temporary state in the dashboard is that we won't be able to differentiate if we click logout or not from the previous page.